massa-execution-worker/src/tests/scenarios_mandatories.rs

```diff
@@ -165,6 +116,36 @@ fn test_nested_call_gas_usage() {
     );
     controller.update_blockclique_status(finalized_blocks.clone(), Default::default());
     std::thread::sleep(Duration::from_millis(10));
+    let bytecode_sub_contract_len = 9367;
```

Comment:
Avoid the hardcoding here?

Reply:
It's in tests and it's impossible (as far as i know) to get the value in the code because it's the len of the inner SC that is included in the bytecode of the parent SC.

Reply:
Added a comment

Reply:
Can't determine from file added an issue #3085